### DISCUSSION TITLE
안녕하세요! 별모양 버튼(구독?), 댓글 관련 문제가 발견되어 공유합니다.

### DISCUSSION THREAD

**Discussion by keepoki** [pronist/hELLO#193] [Q&A] [answered]
안녕하세요! 눈팅만하다가 hELLO 스킨이 너무나 마음에 들어서 적용해보았습니다! 
우선 감사드립니다!! (star 박았습니다!)

### 발견된 문제점

1.  우측 상단에 별모양(구독하기 버튼)이 작동하지 않습니다. (새로고침 됨.. 또는 반응이 없는 경우도 있음) 
![image](https://user-images.githubusercontent.com/26519360/216970016-8c3339f8-72c7-4dee-88d3-0154c54900fc.png)

2.  구글에서 검색을 통해 hELLO님의 블로그 댓글 달기 누르면 반복적으로 로그인한 유저만 가능하다는 alert가 나옵니다..
(이미 저는 로그인 되어있는 데도 인식을 못합니다, 티스토리의 버그인지..)
![image](https://user-images.githubusercontent.com/26519360/216968943-cc07c67a-bef0-4d08-afec-aaf43e55a1af.png)

**Comment by pronist** [comment, marked as answer]
#177 참고 부탁드립니다.
